src/Text/Text.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Text from './Text';
+
+export default {
+  title: 'Components|Text',
+  component: Text,
+};
+
+export const Basic = () => (
+  <>
+    <Text>The basic text component renders as a {'<span />'}</Text>
+    <Text as="h1">It can render as other tags, like this {'<H1/>'}</Text>
+    <Text color="red">It can use colors from the theme</Text>
```

Comment:
Similar to above, should we add a spacing prop case while we're going to the trouble to illustrate these props, or is it similar enough to the existing spacing props on other components, that there's no need for the additional explanation?

Reply:
The latter